envoy/cilium/accesslog.proto

```diff
@@ -34,27 +53,32 @@ message HttpLogEntry {
   // Cilium rule reference
   string cilium_rule_ref = 5;
   
-  // Cilium security ID of the source
+  // Cilium security ID of the source and destination
   uint32 source_security_id = 6;
+  uint32 destination_security_id = 16;
 
   // These fields record the original source and destination addresses,
   // stored in ipv4:port or [ipv6]:port format.
   string source_address = 7;
   string destination_address = 8;
 
+  oneof l7 {
+    HttpLogEntry http = 100;
+  }
+
+  //
+  // Deprecated HTTP fields
+  //
+  Protocol http_protocol = 2;
```

Comment:
Add the `deprecated` option to every field here, cf. https://developers.google.com/protocol-buffers/docs/proto3, e.g.:
```
HttpProtocol http_protocol = 2 [deprecated=true];
```

Reply:
OK, thanks for the pointer!